From a0136f24f788e8bfc4ab74a647b27e115a25d9fb Mon Sep 17 00:00:00 2001
From: Paul Spooren <mail@aparcar.org>
Date: Wed, 19 May 2021 00:55:21 +0200
Subject: [PATCH] nslookup: mention QUERY_TYPE SRV

SRV lookups are supported since "6b4960155 nslookup: implement support
for SRV records" and should therefore be mentioned as a possible
QUERY_TYPE in the help message.

Signed-off-by: Paul Spooren <mail@aparcar.org>
---
 networking/nslookup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/networking/nslookup.c b/networking/nslookup.c
index dda22de0e..2ae8d391c 100644
--- a/networking/nslookup.c
+++ b/networking/nslookup.c
@@ -25,7 +25,7 @@
 //usage:#define nslookup_full_usage "\n\n"
 //usage:       "Query DNS about HOST"
 //usage:       IF_FEATURE_NSLOOKUP_BIG("\n")
-//usage:       IF_FEATURE_NSLOOKUP_BIG("\nQUERY_TYPE: soa,ns,a,"IF_FEATURE_IPV6("aaaa,")"cname,mx,txt,ptr,any")
+//usage:       IF_FEATURE_NSLOOKUP_BIG("\nQUERY_TYPE: soa,ns,a,"IF_FEATURE_IPV6("aaaa,")"cname,mx,txt,ptr,srv,any")
 //usage:#define nslookup_example_usage
 //usage:       "$ nslookup localhost\n"
 //usage:       "Server:     default\n"
-- 
2.30.2

